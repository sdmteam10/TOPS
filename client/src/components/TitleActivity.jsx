import React, { Component } from 'react';

class TitleActivity extends Component {
    state = {

    }

    render() {
        return (
          <div className="bg-primary" style={{ height: '150px' }}>
            <div className='text-light mx-auto text-center'>
              <h1>Activity Scheduler</h1>
              <h6> Select goals or drag goals onto a card to create your own scheduler.</h6>
              <h6> Make up your own activity scheduler.</h6>
            </div>
          </div>
        )
    }
}

export default TitleActivity