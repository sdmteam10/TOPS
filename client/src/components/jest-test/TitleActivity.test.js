// TitleActivity.test.js
import React from 'react'
import TitleActivity from '../TitleActivity'
import renderer from 'react-test-renderer';
import { fireEvent, render } from "@testing-library/react"

//it.only() "only test this <it>."
//it.skip() "not test this <it>."
// done() under expect(){} -  should be only used for async functions

  describe('Test <TitleActivity /> component', function() {
    it('Should render', function() {  
      //Render component should be successed.
      const component = renderer.create(
        <TitleActivity />
      );
        let testSnap = component.toJSON();
        expect(testSnap).toMatchSnapshot(); 
    })
  })