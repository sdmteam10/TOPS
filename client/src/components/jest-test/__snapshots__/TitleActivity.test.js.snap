// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <TitleActivity /> component Should render 1`] = `
<div
  className="bg-primary font-monospace"
  style={
    Object {
      "height": "auto",
    }
  }
>
  <div
    className="mx-auto text-center"
  >
    <br />
    <h2
      className="text-light"
    >
      Activity Scheduler
    </h2>
    <br />
    <h6
      className="text-light"
    >
       Drag activities onto a card to create your own scheduler.
    </h6>
    <h6
      className="text-light"
    >
       Select goals to see activities.
    </h6>
    <h6
      className="text-light"
    >
       Make your own activity scheduler.
    </h6>
    <br />
  </div>
</div>
`;
